File: src/main/java/moe/giga/discord/commands/WhoAmICommand.java
package moe.giga.discord.commands;

import moe.giga.discord.annotations.CommandInfo;
import moe.giga.discord.contexts.MessageContext;

@CommandInfo(name = "whoami", hidden = true)
public final class WhoAmICommand extends Command {

    @Override
    public void onCommand(MessageContext MC, String[] args) {
        MC.sendFormattedMessage("You are **%s**", MC.getUC().getHumanRole());
    }
}

File: src/main/kotlin/moe/giga/discord/commands/WhoAmICommand.kt
package moe.giga.discord.commands

import moe.giga.discord.contexts.MessageContext

@Suppress("unused")
class WhoAmICommand : Command {
    override val name = "whoami"
    override val hidden = true

    override fun execute(MC: MessageContext, args: List<String>) {
        MC.sendMessage("You are **${MC.userCtx.humanRole}**").queue()
    }
}
